src/Fable.Cli/CHANGELOG.md

```diff
@@ -25,8 +27,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### All
 
 * Overall performance improvements
-    * [GH-3620](https://github.com/fable-compiler/Fable/pull/3620) Removed double-dictionary lookups (by @Thorium)
-    * [GH-3624](https://github.com/fable-compiler/Fable/pull/3624) Add G-Research analyzers and fix reported issues (by @nojaf)
+  * [GH-3620](https://github.com/fable-compiler/Fable/pull/3620) Removed double-dictionary lookups (by @Thorium)
+  * [GH-3624](https://github.com/fable-compiler/Fable/pull/3624) Add G-Research analyzers and fix reported issues (by @nojaf)
```

Comment:
Can we please keep the 4 spaces indentation?

There are some other elements below using 4 spaces.

Reply:
Ok, perhaps we can define some markdownlint rules for the file. Currently there's multiple issues e.g: 
<img width="833" alt="Screenshot 2023-12-12 at 09 37 57" src="https://github.com/fable-compiler/Fable/assets/849479/0444f90b-fccf-4a0c-aabf-1d9fd56efddb">

Reply:
We could but nothing guarantee that everyone is going to use VSCode and see the linter information.

It seems like the extension you are using also has a CLI, so to enforce it we could make a pre-commit hooks to enforce CHANGELOG.md formatting.

Is this a good direction to go with? For Fable repositiory,

I came up with the following configuration:

```json
{
    "MD024": false, // Allow duplicates headings
    "MD007": {
        "indent": 4 // Use 4 spaces indentations
    },
    "MD038": false, // Allow space in code snippet, no idea why this is on by default
    "MD013": false // Disable line-length checks we are now using URLs in the changelog and that increase the line length considerably (but make links works on external websites like NuGet, etc)
}
```